docs/r2/hyperparameter_tuning_with_hparams.ipynb

```diff
@@ -0,0 +1,599 @@
+{
```

Comment:
<p>Should you clear outputs?</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/1926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
This is necessary for now given the discussion [here](https://github.com/tensorflow/tensorboard/pull/1909#discussion_r260850621)